dydx/chain.json

```diff
@@ -182,7 +186,11 @@
       }
     ],
     "rest": [
-      {
+	  {
```

Comment:
Can you fix the formatting on this and the following lines please? Also please use an appropriate title for PRs - like "Update Pro Delegators Endpoints"

Reply:
Sorry for the formatting issue, i fix it in the last commit, with a better message too.

Reply:
The formatting was still off. I went ahead and fixed it so you can review the changes for next time